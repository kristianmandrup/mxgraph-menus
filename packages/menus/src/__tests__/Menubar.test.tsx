import React from "react";
import { MenubarComponent } from "./MenubarComponent";
import renderer from "react-test-renderer";

describe("Menubar", () => {
  it("is a menubar", () => {
    const component = renderer.create(<MenubarComponent></MenubarComponent>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
