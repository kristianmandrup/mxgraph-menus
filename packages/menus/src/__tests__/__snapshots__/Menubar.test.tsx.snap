// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menubar is a menubar 1`] = `<div />`;
